src/WebJobs.Script/Utility.cs

```diff
@@ -824,6 +826,13 @@ public static void LogAutorestGeneratedJsonIfExists(string rootScriptPath, ILogg
             // If we dont find the .autorest_generated.json in the function app, we just don't log anything.
         }
 
+        public static void AccumulateDuplicateHeader(HttpContext httpContext, string headerName)
+        {
+            // Add duplicate http header to HttpContext.Items. This will be logged in ResponseContextItemsCheckMiddleware
```

Comment:
Nit -- I think you just removed this middleware in the comment :-)

Reply:
Will fix, thanks